### ISSUE TITLE
Edit preview by drag

### ISSUE THREAD

**Issue by Kitenite** [XD2Sketch/gpt-react-designer#4]
Hello,
It would be great if the styles can be edited by dragging and dropping. I might pick this up but just opening an issue for tracking :)

**Comment by kgoedecke** [comment]
@Kitenite love this idea. I was wondering if we can maybe incorporate something like https://github.com/prevwong/craft.js or any other react based editor. Did you have any preference?

**Comment by Kitenite** [comment]
That looks really good. I’ll try it out!

**Comment by Kitenite** [comment]
Went down the rabbit hole and found [Utopia](https://github.com/concrete-utopia/utopia) which implements a lot of these desired functionalities. Super cool

**Comment by kgoedecke** [comment]
@Kitenite that looks really cool. I'm wondering if we should keep it rather simple for now to have a scope that's achievable. By that I mean limiting the things you can modify to certain thingy like text size, color maybe margin and padding. I definitely think this is something achievable and would help a lot of people write code.

**Comment by Kitenite** [comment]
Yes, I agree a small subset of those would be immensely useful. I'm trying to figure out now how to edit the content inside the iframe even with craft.js since the content is basically just a react string.
